### ISSUE TITLE
My saved tabs are removed!

### ISSUE THREAD

**Issue by BostonLobster** [cnwangjie/better-onetab#219]
扩展版本 (extension version): 1.4.5
浏览器版本 (browser version): Chromium 74.0.3729.169
操作系统 (operating system): Windows 10
问题描述 (problem description):
Problems with the pinned better-onetab management tab.

实际行为 (actual behavior): My tabs are unwanted removed.

期望行为 (expected behavior): The saved tabs should be synced between all opened better-onetab management tab.

重现步骤 (reproduce step):
Reproduce steps:
1. Pin the better-onetab management tab in one window.
2. Open another window, and open some tabs. Click `save all opened tabs in this window`.
3. After step 2, a better-onetab mangement tab will be popped up in that window, but the saved tabs are not shown.
4. However,those saved tabs in step 2 are visible in the pinned tab in step 1.
Then comes the wierd thing:
5. Close the pinned management tab in step 1, and reopen better-onetab management tab. The saved tabs in step 2 are disappear!!!

**Comment by cnwangjie** [comment]
Thank you for your report. I've published a new version. Please reference [https://github.com/cnwangjie/better-onetab/issues/121#issuecomment-519410965](https://github.com/cnwangjie/better-onetab/issues/121#issuecomment-519410965)

If your problem still exists or there is no relation to this problem, please submit a new issue or reopen this issue.

**Comment by VelocityRa** [comment]
I added a bunch of tabs yesterday, and today it lost them again...
